roles/pulp_repository/defaults/main.yml

```diff
@@ -12,6 +12,14 @@ pulp_repository_rpm_repos: []
 pulp_repository_sync_retries: 3
 
 pulp_repository_container_repos_sync_retries: "{{ pulp_repository_sync_retries }}"
+pulp_repository_container_repos_remotes_retries: "{{ pulp_repository_sync_retries }}"
+pulp_repository_container_repos_repositories_retries: "{{ pulp_repository_sync_retries }}"
 pulp_repository_deb_repos_sync_retries: "{{ pulp_repository_sync_retries }}"
+pulp_repository_deb_repos_remotes_retries: "{{ pulp_repository_sync_retries }}"
+pulp_repository_deb_repos_repositories_retries: "{{ pulp_repository_sync_retries }}"
 pulp_repository_python_repos_sync_retries: "{{ pulp_repository_sync_retries }}"
+pulp_repository_python_repos_remotes_retries: "{{ pulp_repository_sync_retries }}"
+pulp_repository_python_repos_repositories_retries: "{{ pulp_repository_sync_retries }}"
 pulp_repository_rpm_repos_sync_retries: "{{ pulp_repository_sync_retries }}"
+pulp_repository_rpm_repos_remotes_retries: "{{ pulp_repository_sync_retries }}"
+pulp_repository_rpm_repos_repositories_retries: "{{ pulp_repository_sync_retries }}"
```

Comment:
Will remove the repos_ prefix

Reply:
> Will remove the repos_ prefix

In two minds, in some ways the repos_ prefix is more consistent. Let me know what you think.